pkg/schema/schema.go

```diff
@@ -435,11 +435,11 @@ func GetSchemaFromComment(comment string) (Schema, string, error) {
 			continue
 		}
 		if insideSchemaBlock {
-			content := strings.TrimPrefix(line, CommentPrefix)
-			rawSchema = append(rawSchema, strings.TrimPrefix(content, CommentPrefix))
+			content := strings.TrimLeft(strings.TrimPrefix(line, CommentPrefix), " ")
```

Comment:
@dadav should we rather use `string.Trim` instead of `TrimLeft` on those 3 places?

Reply:
I think TrimLeft is fine.